web/src/features/fireWeather/components/graphs/plotlyHelper.ts

```diff
@@ -170,9 +170,9 @@ export const populateGraphDataForTempAndRH = (
     }
 
     // From forecast min & max
-    if (tmp_dp_min && tmp_dp_max) {
-      tempDewpointMinMaxDates.push(date)
-      tempDewpointMinMaxValues.push([tmp_dp_min, tmp_dp_max])
+    if (tmp_min && tmp_max) {
```

Comment:
Is it considered an error if either or both of these values are missing? Or should it be logged for transparency?

Reply:
They are actually being used if you time travel to the time where forecasts are recorded. The toggles are disabled if there is no data to display, so it depends on the situation. 
Check this out: https://wps-web-prod.pathfinder.gov.bc.ca/morecast?codes=322&toi=2020-09-10T10:31:00-08:00